game-core/src/main/java/games/strategy/engine/data/RepairRule.java

```diff
@@ -4,6 +4,9 @@
 
 import games.strategy.util.IntegerMap;
 
+/**
+ * A repair rule.
```

Comment:
Restarting class name, describing the scope or responsibility of this class, what it represents, what is a 'repair rule' would be good information to have.